src/reducers/index.js

```diff
@@ -12,9 +13,17 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {
+      ...state,
+      count:++state.count
+    }
+    
     case DECREMENT:
-    // Fill in the body of this case
+    return {
+     ...state,
+     count:--state.count
+    }
```

Comment:
Don't use the `--` operator for updating state values because it will mutate the `state` object first (to be more specific about what happens here, `state.count` is decremented by 1, and then `state.count` is assigned to itself).

Derive a new value from `state.count` instead when you return the `state` object, like so:
```
return {
  ...state,
  count: state.count - 1
};
```